build/azDevOps/azure/air-infrastructure-data-vars.yml

```diff
@@ -93,7 +92,6 @@ variables:
   # Static definition within Influx Config
   - name: DASHBOARD_MEASUREMENT
     value: deployment
-  - group: amido-stacks-influx-credentials-prod
```

Comment:
what about removing from `build/azDevOps/azure/air-infrastructure-data-vars-template.yml`?

Reply:
<img width="714" alt="image" src="https://github.com/amido/stacks-azure-data/assets/131348498/c1b29ffc-7a86-4ea0-a427-9c9e5907c023">

Reply:
updated, keeping the template files for now